api/app/src/v2/Registration/Controller/LayDeputyshipUploadController.php

```diff
@@ -17,9 +16,8 @@ class LayDeputyshipUploadController
 {
     public function __construct(
         private DataCompression $dataCompression,
-        private LayDeputyshipDtoCollectionAssemblerFactory $factory,
-        private LayDeputyshipUploader $uploader,
-        private LoggerInterface $verboseLogger
+        private CSVDeputyshipProcessing $csvProcessing,
+        private LoggerInterface $logger
```

Comment:
I think we have to keep the `$verboseLogger` variable to let the controller know to use our verbose logger service.

I think we have to use the verbose logger otherwise `notice()` won't work on prod like environments.

Reply:
That would be defined in the service.yml & not via a var name. There is no classes referenced in the system as `verbose logger`  The only thing remotely close to this, is `verbose_log_level` which is set to `notice` so long as we are sending our logs via the `notice` level then it's not an issue.

I have just double checked some of the changes I made after refactoring these classes. Seems I changed them from notice to info, so will revert those back.